.storybook/preview.tsx

```diff
@@ -71,9 +73,11 @@ export const parameters: Parameters = {
       date: /Date$/,
     },
   },
+  date: mockDate,
 };
 
 export const decorators: Decorator[] = [
+  mockDateDecorator,
```

Comment:
this is just to stop the visual regression tests from failing because the text is different because the date is different